Formula/abyss-explorer.rb

```diff
@@ -5,6 +5,7 @@ class AbyssExplorer < Formula
   url "https://github.com/bcgsc/ABySS-explorer/releases/download/v2.1.0/ABySS-explorer-2.1.0.tar.gz"
   sha256 "3f22d6d2a0bc5127453d6c9b7b6ddb609d602a3495384f73e603fe6b2967b3a7"
   license "GPL-3.0"
+  revision 1
```

Comment:
```suggestion
```
Removing the revision, because I need to merge this PR without updating bottles.